src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "tracing"
+excerpt: "k6 Tracing experimental API"
+---
+
+<ExperimentalBlockquote />
+
+
+With this experimental module, you can _instrument_ you HTTP requests so that they emit traces as the test runs.
+
+
+| Class/Function                                                           | Description                                                                                                               |
+| :----------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [instrumentHTTP](/javascript-api/k6-experimental/tracing/instrumenthttp) | instruments the k6 http module with tracing capabilities.                                                                 |
+| [Client](/javascript-api/k6-experimental/tracing/client)                 | configurable Client that exposes instrumented HTTP operations and allows selectively instrumenting requests with tracing. |
+| [Options](/javascript-api/k6-experimental/tracing/options)               | Options to configure the tracing instrumentation behavior.                                                         |
+
+## Example
+
+This example demonstrates how to use the tracing API to instrument every HTTP request made in a script with tracing information.
+
+<CodeGroup labels={["example-tracing-module.js"]} lineNumbers={[]} showCopyButton={[true]}>
+
+```javascript
+import { check } from "k6";
+import tracing from "k6/experimental/tracing";
+import http from "k6/http";
+
+// instrumentHTTP will ensure that all requests made by the http module
+// from this point forward will have a trace context attached.
+//
+// The first argument is a configuration object that
+// can be used to configure the tracer.
+//
+// Currently supported HTTP methods are: get, post, put, patch, head,
```

Comment:
nit: asyncRequest is not mentioned. I am not certain if we should have all of them listed as well :shrug:

Reply:
I forgot 🤦🏻 Also think you're right, we could explicitly mention the ones who aren't instead (only `batch`) 👍🏻

Reply:
This seems to not have been fixed :thinking:

Reply:
It was fixed but trhre is a second copy see https://github.com/grafana/k6-docs/pull/1020/files#r1102867581

Reply:
Indeed, I've forgotten it 👍🏻